src/mumble/PasswordDialog.h

```diff
@@ -0,0 +1,26 @@
+#ifndef PASSWORDDIALOG_H
+#define PASSWORDDIALOG_H
+
+#include <QDialog>
+
+namespace Ui {
+class PasswordDialog;
+}
+
+class PasswordDialog : public QDialog {
+	Q_OBJECT
+
+public:
+	explicit PasswordDialog(QWidget *parent = nullptr);
```

Comment:
I think you want to use ``Q_DISABLE_COPY(PasswordDialog)`` instead of an ``explicit`` constructor

Reply:
These two constructs do different things. An explicit ctor avoids implicit conversions and disabling copy prevents (accidental) copies. These two things are orthogonal to each other.